.github/workflows/build.yml

```diff
@@ -16,6 +16,7 @@ jobs:
     env:
       PLATFORM_USER_NAME: ${{ secrets.PLATFORM_USER_NAME }}
       PLATFORM_USER_PASSWORD: ${{ secrets.PLATFORM_USER_PASSWORD }}
+      ORG_ID: ${{ secrets.ORG_ID }}
```

Comment:
Fine by me to use secrets as 'easy to update' config.

But let's not lure ourselves, this bit (the org id) ain't secure at all: we log it in the initial-login logs for e.g. :P

Reply:
haha. I know. it's just to avoid hardcoding it in the tests